src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -27,6 +30,32 @@ const MainLayout = async (props: Props) => {
     return <SignInButton autoSignIn />;
   }
 
+  const accountId = session?.user?.subscriber?.fxa_uid;
+
+  let userId = "";
+  if (accountId && typeof accountId === "string") {
+    // If the user is logged in, use a UUID based on the user's subscriber ID.
+    // TODO determine if we can collect the FxA UID directly https://mozilla-hub.atlassian.net/browse/MNTOR-2180
+    if (process.env.NIMBUS_UUID_NAMESPACE) {
+      userId = uuidv5(accountId, process.env.NIMBUS_UUID_NAMESPACE);
+    } else {
+      console.error("NIMBUS_UUID_NAMESPACE env var not set");
+    }
+  }
+
+  if (!userId) {
+    console.error("No user ID for Nimbus telemetry");
+  }
+
+  try {
+    // TODO For initial A/A testing `features` is unused. https://mozilla-hub.atlassian.net/browse/MNTOR-2182
+    const features = await getExperiments(userId);
```

Comment:
I know this is still early fetch-and-log code, but do we have a way to force a user into an experiment?
For example, if Softvision or us are testing flows, is there a special flag or ripcord we can pull to force ourselves into a certain state for manual/automated testing?
Acceptable answers include: "yes", "no", "not yet", "what?", `<other>`, or colorful profanity if your choosing.

Reply:
We do have allow-lists for the current feature-flag implementation - you can use the UI at `/redesign/admin/feature-flags`. However that works by storing flag state and allow-list in the database, so would work for this "force a user into experiment" use case, however we should add more specific support for Nimbus (there's no need to use the database for this, it'll just be for the current session) https://mozilla-hub.atlassian.net/browse/MNTOR-2182